test/sequential/rewardsv2.poolLiquidityRewards.test.ts

```diff
@@ -205,6 +216,46 @@ describe("rewards v2 tests", () => {
         assetAmount.divn(2)
       );
     });
+    test("Given a user with Liquidity activated When tries to mint some more Then the user activated amount will grow on that value", async () => {
+      const rewardsInfoBefore = await getRewardsInfo(
+        testUser3.keyRingPair.address,
+        liqId
+      );
+
+      await mintLiquidity(
+        testUser3.keyRingPair,
+        MGA_ASSET_ID,
+        secondCurrency,
+        defaultCurrencyValue
+      );
+
+      const rewardsInfoAfter = await getRewardsInfo(
+        testUser3.keyRingPair.address,
+        liqId
+      );
+
+      const rewardsDifference = rewardsInfoAfter.activatedAmount.sub(
+        rewardsInfoBefore.activatedAmount
+      );
+
+      expect(rewardsDifference.div(new BN(100))).bnEqual(
+        defaultCurrencyValue.div(new BN(100))
+      );
+    });
+    test("Given a user with bonded but not activated liq tokens WHEN he tries to activate THEN the tokens are activated for rewards", async () => {
```

Comment:
@AlexChetverov  where the bond happen? can not see it :upside_down_face:

Reply:
Bonded tokens are tokens reserved when a user joins as candidate. You can "simply" 

1. Sudo ->  staking.enableLiquiditytoken
2.- User -> staking.joinAsCandidate 
--> Expected: Tokens are reserved and in MPL pallet, the tokens appear as: stakedButUnactivatedTokens
I think there are some examples in the code, but feel free to reach me if you need some help :)